ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -425,6 +425,13 @@
     reason="Test will not run on Fusion HCI provider and Client clusters",
 )
 
+skipif_hci_provider_or_client = pytest.mark.skipif(
```

Comment:
Maybe this should replace skipif_hci_provider_and_client. I am not sure in what case we would skip if provider and client are present and not if only one is present.

Reply:
I don't want to replace it in this PR. PR is XXL and it is to costly to run additional tests to UI suit to verify all tests work as expected. Also will be too hard to separate this XXL PR with makr - related issues in case we would think to pull changes back